rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/config/configured_rule.php

```diff
@@ -54,5 +57,10 @@
         // special case with following comment becoming a inner value
         new AnnotationToAttribute('When', When::class, useValueAsAttributeArgument: true),
         new AnnotationToAttribute('Then', Then::class, useValueAsAttributeArgument: true),
+
+        // simple tag to attribute
+        new AnnotationToAttribute('OldName1', NewName1::class),
+        new AnnotationToAttribute('OldName2', NewName2::class),
+        new AnnotationToAttribute('SameName', SameName::class),
```

Comment:
this should have `useValueAsAttributeArgument: true`  like `@When` or `@Then` above it

Reply:
that's better, which means needs patch on the option vs no argument at all